### ISSUE TITLE
Document should have one main landmark

### ISSUE THREAD

**Issue by hkier** [chihacknight/ghost-buses-frontend#89]

**Comment by hkier** [comment]
I have run a light house report against the website.  This was not reported as an accessibility issue.
